test_project/views/frontend/widget_data.py

```diff
@@ -20,7 +22,7 @@ def __init__(self):
         ]
 
     def update_state(self):
-        self.server_state.set_text(dumps(self.data))
+        self.server_state.set_text(dumps(self.widget_data))
```

Comment:
I am not sure if this test view is running correctly. From what the view tells me the ``client-state``-node should always contain the same value as the server side?
I don't see that behavior here. The  ``client-state ``-div is always empty.

Reply:
Ack! Fixed